content/en/docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -62,6 +62,11 @@ and respecting the PodDisruptionBudget you have defined). It is then safe to
 bring down the node by powering down its physical machine or, if running on a
 cloud platform, deleting its virtual machine.
 
+{{< note >}}
+[nodeName](/docs/concepts/scheduling-eviction/assign-pod-node/#nodename) bypasses the scheduler, 
+thus evicted Pods will still run on a drained node.
```

Comment:
Try this:
```suggestion
If any new Pods tolerate the `node.kubernetes.io/unschedulable` taint, then those Pods
might be scheduled to the node you have drained. Avoid tolerating that taint other than
for DaemonSets.

If you or another API user directly set the [`nodeName`](/docs/concepts/scheduling-eviction/assign-pod-node/#nodename)
field for a Pod (bypassing the scheduler), then the Pod is bound to the specified node
and will run there, even though you have drained that node and marked it unschedulable.
```

```

Reply:
I probably wouldn't wrap this in a note. These paragraphs describe the normal operation of Kubernetes; they are worth stating, but the callout about ignoring system pods (see line 51) is more important.

Reply:
This revision didn't answer my question. If I have a Pod with `nodeName` set to to `"nodeA"`, what happens if I'm now draining the "nodeA" node? My Pod is not affected at all, or it will be killed and then "rescheduled" to the same node and run there?